app/scripts/modules/amazon/src/loadBalancer/configure/common/SecurityGroups.tsx

```diff
@@ -193,7 +193,11 @@ class SecurityGroupsImpl extends React.Component<ISecurityGroupsProps, ISecurity
                     The following {FirewallLabels.get('firewalls')} could not be found in the selected
                     account/region/VPC and were removed:
                   </p>
-                  <ul>{removed.map(sg => <li key={sg}>{sg}</li>)}</ul>
+                  <ul>
+                    {removed.map(sg => (
+                      <li key={sg}>{sg}</li>
+                    ))}
+                  </ul>
```

Comment:
stop it prettier.

Reply:
Is it possible your prettier is out of date?  I'm going to re-run current prettier across the entire codebase next.